src/tokens/hooks.ts

```diff
@@ -44,12 +41,12 @@ export function useTokensWithUsdValue(networkIds: NetworkId[]) {
 }
 
 export function useTotalTokenBalance() {
-  const supportedNetworkIds = getSupportedNetworkIdsForTokenBalances()
+  const supportedNetworkIds = getSupportedNetworkIds()
   return useSelector((state) => totalTokenBalanceSelector(state, supportedNetworkIds))
```

Comment:
should we eventually drop the networkId option from selectors and do this at the top level selector or even filter them out after getting response from the API?

Reply:
yes absolutely, i didn't want to include that here because the PR would become even bigger